app/main.py

```diff
@@ -33,4 +33,17 @@ def extract_text(url: HttpUrl = Query(..., description="The URL to extract text
 
     soup = BeautifulSoup(response.content, "html.parser")
     text = soup.get_text(separator="\n", strip=True)
-    return {"text": text}
+
+    # Extract images from various tags
+    images = []
```

Comment:
Put this in an utility method

Reply:
Thank you for the feedback. I will refactor the code to move the image extraction logic into a utility method. This will make the code cleaner and more modular. I'll update the PR accordingly.